druid-shell/src/mac/mod.rs

```diff
@@ -13,7 +13,9 @@
 // limitations under the License.
 
 //! macOS implementation of window creation.
-#![allow(non_snake_case)]
+
+#![allow(non_snake_case,
+    clippy::let_unit_value,)]
```

Comment:
This is a pretty classic example of why I haven't been a fan of clippy. In this case, it's directly telling us to do the wrong thing, because it doesn't have domain understanding of objC.

Reply:
Agreed, but also fair enough. `objc` is a weird case.